frontend/src/components/Configuration/EnvironmentRegionSetting.jsx

```diff
@@ -3,26 +3,23 @@ import Grid from '@mui/material/Grid';
 import InputLabel from '@mui/material/InputLabel';
 import MenuItem from '@mui/material/MenuItem';
 import { TextField } from '@mui/material';
-import Tooltip from '@mui/material/Tooltip';
-import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
-import Stack from '@mui/material/Stack';
-import { TimePicker } from '@mui/x-date-pickers/TimePicker';
-import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import { StyledSelect } from '../../css/SimulationPageStyles';
 import { ENVIRONMENT_ORIGINS, ENVIRONMENT_ORIGIN_VALUES } from '../../utils/const';
 import { EnvironmentModel } from '../../model/EnvironmentModel';
 import PropTypes from 'prop-types';
+import TimeGridComponent from './TimeGridComponent';
 
 
 const EnvironmentRegionSetting = ({ envConf, setEnvConf }) => {
 
+
     const handleRegionBasedPropSetting = (val) => {
         if (val.target.value != "Specify Region") {
             ENVIRONMENT_ORIGIN_VALUES.map(obj => {
                 if (obj.value == val.target.value) {
-                    envConf.setOriginLatitude(obj.Latitude);
-                    envConf.setOriginLongitude(obj.Longitude);
-                    envConf.setOriginHeight(obj.Height);
+                    envConf.setOriginLatitude(obj.latitude);
+                    envConf.setOriginLongitude(obj.longitude);
+                    envConf.setOriginHeight(obj.height);
```

Comment:
@LaxminarayanaV7416 There is a typo here in above 3 lines. They are defined in Caps in `ENVIRONMENT_ORIGIN_VALUES`

Reply:
No No that is not typo, I have changed the JSON value of origin in model you can check that, because of that it wasnt reflecting in the starting, this should be present

Reply:
@LaxminarayanaV7416 I am aware that you updated the property names in EnvironmentModel.Origin. But here `obj` is an item from `ENVIRONMENT_ORIGIN_VALUES` array but not from updated model